packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -1,40 +1,51 @@
 const pMemoize = require("p-memoize");
 
+/**
+ * Convenient helper for calling Booking API flow functions like C1, C2, B, etc
+ * and caching the results.
+ *
+ * For each of these functions, the output is cached the first time it is run
+ * and this cached value is used for subsequent calls.
+ *
+ * Additionally, flow functions call any dependent functions (e.g. B calls C2,
+ * which calls C1, etc).
+ */
 class FlowHelper {
   constructor (helper) {
     this.state = helper;
   }
 
-  getDatasetSite = pMemoize(async getDatasetSite => {
+  getDatasetSite = pMemoize(async () => {
```

Comment:
I checked all the uses of these functions, and none of them supply an arg to any of these flow functions. And these args are not used within the function itself. Therefore, they are useless. I suspect they were introduced as a misunderstanding of async arrow function syntax

Reply:
There used to be a thing about naming functions so they could be easily recognised in a stack trace...? Not sure if that guidance is now out-of-date? If that's not it this looks great!

Reply:
> There used to be a thing about naming functions so they could be easily recognised in a stack trace...?

The above doesn't name the function. It acts as the first agument to the function. Node.js stack traces for promises and arrow functions are much better than they used to be, so I believe `getSatasetSite` would be picked up by stack traces though I'm not completely sure.

Reply:
sorry you're totally right! in that case lgtm!